examples/helloworld/client.py

```diff
@@ -134,8 +134,8 @@ def _as_dict(data: typing.Any, *, convert_all: bool = True) -> dict[str, typing.
 
 def _convert_transaction_parameters(
     transaction_parameters: algokit_utils.TransactionParameters | None,
-) -> algokit_utils.CommonCallParametersDict:
-    return typing.cast(algokit_utils.CommonCallParametersDict, _as_dict(transaction_parameters))
+) -> algokit_utils.TransactionParameters:
```

Comment:
What's the difference between these types?

Reply:
@robdmoore left a comment for Dan to clarify